tests/lifecycle/tests/lifecycle_pod_owner_type.go

```diff
@@ -115,8 +123,7 @@ var _ = Describe("lifecycle lifecycle-pod-owner-type", func() {
 	})
 
 	// 47430
-	It("Multiple deployments, 1 pod not related to any resource [negative]", func() {
-
+	It("Two deployments, 1 pod not related to any resource [negative]", func() {
```

Comment:
Please select either Two or 2 like in the previous test case

Reply:
done, thanks.